src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/mha_wo_transpose.cpp

```diff
@@ -150,6 +150,35 @@ INSTANTIATE_TEST_SUITE_P(
                        ::testing::Values(CPUTestUtils::cpu_bf16_plugin_config)),
     MHA::getTestCaseName);
 
+
+INSTANTIATE_TEST_SUITE_P(
```

Comment:
Minor: it seems like we can easily combine 3D and 4D instances. May I ask you to do this?

Reply:
Right, combined them. Thanks!